app/templates/main/browse.html

```diff
@@ -335,59 +335,56 @@ <h1 class="govuk-heading-l browse__records-found__text">Records found {{ num_rec
           {% if num_records_found > 0 %}
             <dl class="govuk-summary-list browse-grid__list">
               <div class="govuk-summary-list__row govuk-summary-list__row__browse-grid-list">
-                <dt class="govuk-summary-list__key browse-grid__key browse-grid__key__transferring-body">
+                <dt class="govuk-summary-list__key govuk-summary-list__key__left">
```

Comment:
had to do this to make the columns align properly. Not quite sure why we wanted to right align them previously anyway